mex/model/entities/bibliographic-resource.json

```diff
@@ -0,0 +1,416 @@
+                "$ref": "/schema/entities/person#/identifier"
+            },
+            "sameAs": [
+                "http://purl.org/dc/terms/contributor"
+            ],
+            "type": "array"
+        },
+        "hadPrimarySource": {
+            "$ref": "/schema/entities/primary-source#/identifier",
+            "description": "A primary source for a topic refers to something produced by some agent with direct experience and knowledge about the topic, at the time of the topic's study, without benefit from hindsight. Because of the directness of primary sources, they 'speak for themselves' in ways that cannot be captured through the filter of secondary sources. As such, it is important for secondary sources to reference those primary sources from which they were derived, so that their reliability can be investigated. A primary source relation is a particular case of derivation of secondary materials from their primary sources. It is recognized that the determination of primary sources can be up to interpretation, and should be done according to conventions accepted within the application's domain ([PROV-O, 2013-04-30 ](http://www.w3.org/TR/2013/REC-prov-o-20130430/)).",
+            "sameAs": [
+                "http://www.w3.org/ns/prov#hadPrimarySource"
+            ]
+        },
+        "identifier": {
+            "$ref": "/schema/fields/identifier",
+            "description": "An unambiguous reference to the resource within a given context. Persistent identifiers should be provided as HTTP URIs ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "sameAs": [
+                "http://purl.org/dc/elements/1.1/identifier"
+            ]
+        },
+        "identifierInPrimarySource": {
+            "description": "The identifier of the item used in the primary source.",
+            "examples": [
+                "123456",
+                "item-501",
+                "D7/x4/zz.final3"
+            ],
+            "maxLength": 1000,
+            "minLength": 1,
+            "type": "string"
+        },
+        "isbnIssn": {
+            "default": [],
+            "description": "Either the ISBN (for books) or ISSN (for periodicals) of the publication.",
+            "items": {
+                "examples": [
+                    "ISBN 90-70002-34-5",
+                    "ISBN 90-70002-34-x",
+                    "ISBN 90-70002-34-5x",
+                    "ISBN 90-700-02-34-5",
+                    "ISBN: 978-3-642-11746-6",
+                    "978-3-642-11746-6",
+                    "ISSN 0176-6996",
+                    "ISSN 1430-855X",
+                    "1430-8551",
+                    "1467-9442"
+                ],
+                "type": "string"
+            },
+            "sameAs": [
+                "http://datacite.org/schema/kernel-4/alternateIdentifier"
+            ],
+            "type": "array"
+        },
+        "issue": {
```

Comment:
good point. but I think for "issue" it is not very likely, since it it only relevant for journals and they stick to numbering their issues. but for "volume" it could be possible, since this can be also used for books ("Sammelband"). but maybe it is just safer to change both properties to be valid having also string values?